packages/manager/apps/web/client/app/components/expiration/service-expiration-date.component.html

```diff
@@ -30,7 +30,10 @@
                 </span>
             </div>
         </div>
-        <div data-ng-if="$ctrl.isExpired() || !$ctrl.isInAutoRenew()">
+
+        <div
+            data-ng-if="($ctrl.isExpired() || !$ctrl.isInAutoRenew()) && ($ctrl.serviceInfos.domain !== $ctrl.serviceName)"
```

Comment:
```suggestion
            data-ng-if="($ctrl.isExpired() || !$ctrl.isInAutoRenew()) && $ctrl.serviceInfos.domain !== $ctrl.serviceName"
```
I think that the brackets are not necessary